src/components/BookItem.js

```diff
@@ -1,12 +1,21 @@
+/* eslint-disable camelcase */
 import PropTypes from 'prop-types';
 import '../modules/BookItem.css';
-// eslint-disable-next-line arrow-body-style
+import { useDispatch } from 'react-redux';
+import { removedBook } from '../redux/books/bookSlice';
+
 const BookItem = ({ bookListProp }) => {
+  const dispatch = useDispatch();
+  const handleDeleteBook = (item_id) => {
+    dispatch(removedBook(item_id));
+    console.log(item_id);
+  };
+
   return (
     <li className="book-item-container">
       <p>{bookListProp.title}</p>
       <p>{bookListProp.author}</p>
-      <button type="submit">Delete</button>
+      <button onClick={() => handleDeleteBook(bookListProp.item_id)} type="submit">Delete</button>
     </li>
 
   );
```

Comment:
![image](https://github.com/Ayokunnumi1/bookstore-app-2/assets/93039359/25d78c84-8a0f-43f5-b87a-79f45046e6b7)

- [ ] It would be nice if you fix the highlighted link in the image above 👀 because it is broken 😨 Kindly fix it or remove it **(If you don't have any dedicated section for this link)** to make your readme descriptive and professional 👍